### PR TITLE
feat(user-notification): Document scope

### PR DESCRIPTION
**Pull request by rafnarnason**
replacing scopes in favor of documents scope for service

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Comment by rafnarnason** [comment]
@coderabbitai pause

**Review by kksteini — APPROVED** [review]

**Review by GunnlaugurG — APPROVED** [review]
LGTM✍️

File: apps/service-portal/src/auth.ts [hunk 1]
```diff
@@ -52,8 +51,6 @@ const SERVICE_PORTAL_SCOPES = [
   ApiScope.healthRightsStatus,
   ApiScope.healthDentists,
   ApiScope.signatureCollection,
-  NotificationsScope.read,
-  NotificationsScope.write,
```
Comment (disaerna) [inline apps/service-portal/src/auth.ts:56]: Doesnt service portal need something else instead or is this just being removed completely?

Comment (rafnarnason) [reply apps/service-portal/src/auth.ts:56]: yes -  these are being replaced by DocumentsScope which is already one of the imports

Comment (disaerna) [reply apps/service-portal/src/auth.ts:56]: Ah 😆 ofcourse !

**Review by disaerna — APPROVED** [review]

**Review by eirikurn — APPROVED** [review]

**Pull request merged**